src/gui/plugins/scene_manager/GzSceneManager.cc

```diff
@@ -14,10 +14,15 @@
  * limitations under the License.
  *
 */
+
+#include <map>
 #include <set>
 
+#include "../../GuiRunner.hh"
```

Comment:
is it fine to add relative paths ?

Reply:
I think so. It's was done here as well: https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo6/src/gui/plugins/modules/EntityContextMenu.cc#L18